sdcm/cluster.py

```diff
@@ -4307,6 +4307,16 @@ def _wait_for_nodes_up_and_normal():
 
         _wait_for_nodes_up_and_normal()
 
+    @retrying(n=360, sleep_time=10, allowed_exceptions=ClusterNodesNotReady, message="Waiting for CQLPing to return")
```

Comment:
I wouldn't use the term CQLPing, since it's not an actual something anywhere.

Reply:
re-named